{{cookiecutter.project_slug}}/_/deployment/python/deployment/database/base/service.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: "postgres"
+spec:
+  type: ClusterIP
+  ports:
+  - name: postgresql
+    port: 5432
+    targetPort: postgresql
```

Comment:
This would relate to the comment above, too: Either "postgres" (preferred) or "postgresql" everywhere.